root/utility/sanitizedContext.js

```diff
@@ -21,6 +21,10 @@ function sanitizedContext(
 ) /*: SanitizedCatalystContextT */ {
   const stash = $c.stash;
   const user = $c.user;
+  const genreMap = stash.genre_map;
+  const genreMapStashObject = genreMap
+    ? ({genre_map: genreMap})
+    : {};
```

Comment:
This feels a bit weird, so if there's a better way to combine this with current_language, let me know!